### ISSUE TITLE
Improve OnlineProbe docstring to document default LARS optimizer

### ISSUE THREAD

**Issue by huguesva** [rbalestr-lab/stable-pretraining#346]
## Issue

The `OnlineProbe` callback's docstring has **incorrect documentation** for the `optimizer` parameter.

## Current Docstring (INCORRECT)

```python
optimizer: Optimizer configuration for the probe. Can be:
    - str: optimizer name (e.g., "AdamW", "SGD", "LARS")
    - dict: {"type": "AdamW", "lr": 1e-3, ...}
    - partial: pre-configured optimizer factory
    - optimizer instance or callable
    - None: inherits from main Module's optimizer config (default)  # ❌ FALSE
```

## Actual Behavior

When `optimizer=None`, the callback **does NOT inherit** from the main module. Instead, it uses LARS as a separate optimizer with these settings (from `stable_pretraining/callbacks/utils.py`):

```python
LARS(
    lr=0.1,
    clip_lr=True,
    eta=0.02,
    exclude_bias_n_norm=True,
    weight_decay=0,
)
```

## Problem

This caused confusion during training:
- Main module uses AdamW (from config)
- Linear probe uses LARS (default)
- Docstring claims probe would inherit AdamW
- Logs show two different optimizers, contradicting the documentation

## Suggested Fix

```python
optimizer: Optimizer configuration for the probe. Can be:
    - str: optimizer name (e.g., "AdamW", "SGD", "LARS")
    - dict: {"type": "AdamW", "lr": 1e-3, ...}
    - partial: pre-configured optimizer factory
    - optimizer instance or callable
    - None: uses LARS(lr=0.1, clip_lr=True, eta=0.02, exclude_bias_n_norm=True, 
      weight_decay=0), which is the standard for SSL linear probes (default)
```

The same issue likely applies to the `scheduler` docstring.
